openmdao/core/vec_wrapper_complex_step.py

```diff
@@ -0,0 +1,298 @@
+""" VecWrapper 'wrapper' that is used for component-wise complex step."""
+
```

Comment:
assume you have considered this, but it looks like quite a bit of pass-through code here..  I wonder if just adding this functionality (some attributes and two extra functions?) to the existing classes might be the lesser of two evils?